etc/di.xml

```diff
@@ -34,6 +34,22 @@
     <type name="Magento\Quote\Model\Quote\Config">
         <plugin name="avatax_quote_model_quote_config" type="ClassyLlama\AvaTax\Plugin\Quote\Model\Quote\Config" />
     </type>
+
+    <!--
+    /**
+     * Add plugin and preferences to display tax summary information on frontend
+     * See https://github.com/classyllama/ClassyLlama_AvaTax/issues/70 for details
+     */
+     -->
+    <!--// BEGIN EDIT-->
```

Comment:
Remove begin/end edit comments. Those are only needed when indicating what code has been customized vs _native_ Magento code. This is our custom file, so those comments don't serve any purpose.